src/crystal/event_loop.cr

```diff
@@ -1,28 +1,56 @@
 require "./event"
 
-module Crystal::EventLoop
-  @@eb = Crystal::Event::Base.new
-  @@dns_base : Crystal::Event::DnsBase?
+class Thread
```

Comment:
Could we make this `Crystal::Thread`? Just to hide it more from the public API and also it will play nicely with #7995 which does that too.